README.md

```diff
@@ -31,6 +33,8 @@ npm run dev
 
 Open [http://localhost:3000](http://localhost:3000) with your browser to see the website.
 
+Open [http://localhost:1337/admin/](http://localhost:1337/admin/) with your browser to see the CMS website.
+
```

Comment:
Running `npm run dev` the command automatically opens the browser to strapi. Can we remove this behaviour?

Reply:
🤔 Are you sure? I thought the `admin.autoOpen`[^1] does the trick

[^1]: https://docs.strapi.io/dev-docs/configurations/admin-panel#:~:text=/admin-,autoOpen,-Enable%20or%20disable

Reply:
I had already tried but until admin user isn't setup it does not affect strapi launching browser. I see that exist [this issue](https://github.com/strapi/strapi/issues/5960) that it isn't resolved yet. However i had the `autoOpen` paramiter.